app/javascript/controllers/select-controller.js

```diff
@@ -0,0 +1,11 @@
+import { Controller } from "@hotwired/stimulus";
+
+import TomSelect from "tom-select";
+
+export default class extends Controller {
+  connect() {
+    new TomSelect("#transaction_creditor", {
```

Comment:
Zorgt dit ervoor dat je enkel Tom kan selecteren als creditor? Want daar ben ik misschien niet helemaal akkoord mee...

Reply:
Nee, helaas. TomSelect is een select2/typehead/... alternatief dat geen framework dependency (jquery, react, vue, ...) heeft.